app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptionsViewModel.kt

```diff
@@ -218,6 +232,13 @@ class DebugDataOptionsViewModelImpl
         }
     }
 
+    override fun enableAsyncNotifications(enabled: Boolean) {
+        viewModelScope.launch {
+            startUsingAsyncNotifications()
```

Comment:
Shouldn't `startUsingAsyncNotifications` be called only if `enabled == true`? 🤔

Reply:
Good point actually is not needed the param, since the switch is enabled /observed by the client capability. will fix!

Reply:
fixed !